packages/lib/src/types/treeData.ts

```diff
@@ -11,6 +11,12 @@ export type CategoryNode = {
     childCategories?: Category[];
     infoButtonText?: string[];
     subCategoryName?: string;
+    extra?: CategoryExtra;
+};
+
+export type CategoryExtra = {
+    link?: string;
+    display?: string;
```

Comment:
These are required in the JSON schema so why not make them required here as well?